### PR TITLE
New version: GodotEngine.GodotEngine version 4.2.2

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/149751)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-04-18T01:27:12.7983636Z ##[error]Manifest Validation Failed
 2024-04-18T01:27:17.8841836Z ##[section]Finishing: Validate Manifest

(Automated response - build 876.)

**Comment by vikingnope** [comment]
@Exorcism0666 kindly take a look at my review please

File: manifests/g/GodotEngine/GodotEngine/4.2.2/GodotEngine.GodotEngine.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: GodotEngine.GodotEngine
+PackageVersion: 4.2.2
+InstallerType: zip
+NestedInstallerType: portable
```
Comment (stephengillie) [inline manifests/g/GodotEngine/GodotEngine/4.2.2/GodotEngine.GodotEngine.installer.yaml:7]:
```suggestion
NestedInstallerType: portable
  NestedInstallerFiles:
  - RelativeFilePath: Godot_v4.2.2-stable_win64.exe
    PortableCommandAlias: Godot_v4.2.2-stable_win64.exe
```

**Comment by BiosPlus** [comment]
Interestingly I tried the updated manifest in here and it passed fine on my machines. May need a rerun

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335141
> 2024-04-21 09:09:42.121 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\GodotEngine.GodotEngine.4.2.2\extracted\Godot_v4.2.2-stable_win64.exe

(Automated response - build 877.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335141
> 2024-04-22 11:07:18.968 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\GodotEngine.GodotEngine.4.2.2\extracted\Godot_v4.2.2-stable_win64.exe

(Automated response - build 877.)

**Comment by stephengillie** [comment]
Close with reason: Duplicate of #150735;

**Pull request closed without merging**